Базовая стилизация всех страниц

fesspav opened the pull request:
---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/33/user/1901121/tasks/5)

---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/33/user/1901121/tasks/6)

:boom: https://htmlacademy-htmlcss.github.io/1901121-sedona-33/3/

fesspav committed:
добавил картинки в папку images

картинки оптимизированы

fesspav committed:
3.21 Графика для всех страниц

1. Добавлены все картинки
2. Подключены контентные изображения
3. Добавлены размеры и описание
4. Добавил дополнительные классы
5. Кое-где заменил тег на нумерованный список
6. в layout так и не получилось пока добавить подписку

p.s. файл style.css отправлю в следующем дз

efiand commented:
Замечания и примеры правок находятся [здесь](https://github.com/fesspav/1901121-sedona-33/pull/1/commits/737a275f00d265003fca7b39f78b8f803ba77544)

fesspav committed:
3.22. Базовая стилизация всех страниц

я увлекся. видимо много добавил лишнего и неправильно, но вроде бы основное, что требовалось, учел.
Правки твои тоже постарался все учесть, на коммиты ответил все почти, но есть часть, которые остались непонятыми. кидаю стили и html

fesspav committed:
обновление html и css

обновил html

fesspav committed:
действующие и удаленные картинки
